### ISSUE TITLE
Generate full super-resolution training dataset across models and flow conditions

### ISSUE THREAD

**Issue by MarcosAsh** [MarcosAsh/3dFluidDynamicsInC#154]
The current super-resolution model was trained on a single config (ahmed25, wind=1.0, Re=auto) which produced 27,630 patches at 22 dB PSNR. For the model to generalize, it needs data from all models and flow conditions.

### Full sweep

Using the config in `ml/superres/superres_config.yaml`:
- 3 models (car, ahmed25, ahmed35)
- 6 wind speeds (0.5, 1.0, 1.5, 2.0, 2.5, 3.0)
- 3 Reynolds numbers (0, 5000, 10000)
- Total: 54 configs

At ~27K patches per config, that's ~1.5M patches (~800 MB .srbin file).

### How to run

```bash
python ml/superres/gen_pairs.py --config superres_config.yaml --modal --resume
```

The --modal flag runs simulations on Modal GPU (A10G). The --resume flag skips completed configs, so the sweep can be interrupted and continued.

### Estimated cost

- Each config runs a 64x32x32 fine simulation for 5-15 seconds on an A10G
- ~54 configs at ~2-3 minutes each = ~2-3 hours of GPU time
- At Modal A10G pricing (~$0.36/hr), total cost ~$1-2

### After data generation

1. Retrain: `python ml/superres/train_superres.py --data ml/superres/dataset/patches.srbin --epochs 200`
2. Export: `python ml/superres/export_weights.py --checkpoint ... --output assets/sr_model.bin`
3. Evaluate: check PSNR holds across models, not just ahmed25

Depends on the fine grid resolution being large enough to actually resolve flow features -- see #152 for the Cd accuracy issue that may affect training data quality.
